dataactcore/models/jobTrackerInterface.py

```diff
@@ -1,14 +1,29 @@
 import traceback
 from uuid import uuid4
+from sqlalchemy import func, or_
 from sqlalchemy.orm import joinedload
+from dataactcore.interfaces.db import GlobalDB
 from dataactcore.models.baseInterface import BaseInterface
 from dataactcore.models.jobModels import Job, JobDependency, JobStatus, JobType, Submission, FileType, PublishStatus, FileGenerationTask
+from dataactcore.models.stagingModels import AwardFinancial
 from dataactcore.utils.statusCode import StatusCode
 from dataactcore.utils.responseException import ResponseException
 from dataactcore.utils.cloudLogger import CloudLogger
 from dataactcore.utils.jobQueue import enqueue
 from dataactvalidator.validation_handlers.validationError import ValidationError
 
+def obligationStatsForSubmission(submission_id):
+    sess = GlobalDB.db().session
+    base_query = sess.query(func.sum(AwardFinancial.transaction_obligated_amou)).\
+        filter(AwardFinancial.submission_id == submission_id)
+    procurement = base_query.filter(AwardFinancial.piid != None)
+    fin_assist = base_query.filter(or_(AwardFinancial.fain != None, AwardFinancial.uri != None))
+    return {
+        "total_obligations": float(base_query.scalar() or 0),
```

Comment:
You and @cmc333333 spoke about this, so apologies for being behind, do we need to cast the return value at all here?